src/deploymentmanager.ts

```diff
@@ -343,6 +375,79 @@ export class DeploymentManager implements IDeploymentManager {
+                              'documentDBConnectionString',
+                              'storageAccountName',
+                              'storageAccountKey',
+                              'storageConnectionString',
+                              'messagesEventHubConnectionString',
+                              'messagesEventHubName',
+                              'actionsEventHubConnectionString',
+                              'actionsEventHubName',
+                              'telemetryStorageType',
+                              'tsiDataAccessFQDN',
+                              'office365ConnectionUrl',
+                              'logicAppEndpointUrl',
+                              'azureMapsKey'];
+        outputParams.forEach((paramName) => {
+            if (this._keyVaultParams[paramName] !== undefined && outputs[paramName] !== undefined) {
+                this._keyVaultParams[paramName].value = outputs[paramName].value;
+            }
+        });
+
+        this.setKVParamValue('storageEndpointSuffix', this._azureHelper.getStorageEndpointSuffix());
+        this.setKVParamValue('solutionWebsiteUrl', outputs.azureWebsite.value);
+        this.setKVParamValue('diagnosticsEndpointUrl', answers.diagnosticsEndpointUrl || 'DEFAULT_DIAGNOSTICS_ENDPOINT_URL');
+        this.setKVParamValue('aadAppId', answers.appId);
+        this.setKVParamValue('aadAppSecret', answers.servicePrincipalSecret);
+        this.setKVParamValue('authIssuer', this._azureHelper.getAuthIssuserUrl(answers.aadTenantId));
+        this.setKVParamValue('iotHubName', outputs.iotHubHostName);
+        this.setKVParamValue('subscriptionId', this._subscriptionId);
+        this.setKVParamValue('solutionType', this._solutionType);
+        this.setKVParamValue('applicationSecret', genPassword());
+        this.setKVParamValue('armEndpointUrl', this._environment.resourceManagerEndpointUrl);
+        this.setKVParamValue('aadEndpointUrl', this._environment.activeDirectoryEndpointUrl);
+
+        if (answers.deploymentSku === 'standard') {
+            this.setKVParamValue('authRequired', 'true');
+        } else {
+            this.setKVParamValue('authRequired', 'false');
+        }
+
+        if (answers.deploymentSku !== 'local') {
+            this.setKVParamValue('telemetryWebServiceUrl', 'http://telemetry-svc:9004/v1');
+            this.setKVParamValue('configWebServiceUrl', 'http://config-svc:9005/v1');
+            this.setKVParamValue('iotHubManagerWebServiceUrl', 'http://iothub-manager-svc:9002/v1');
+            this.setKVParamValue('storageAdapterWebServiceUrl', 'http://storage-adapter-svc:9022/v1');
+            this.setKVParamValue('authWebServiceUrl', 'http://auth-svc:9001/v1');
+            this.setKVParamValue('deviceSimulationWebServiceUrl', 'http://device-simulation-svc:9003/v1');
+        }
+    }
+
+    private setKVParamValue(paramName: string, value: any) {
+        if (this._keyVaultParams[paramName] !== undefined) {
```

Comment:
Same here:` if (this._keyVaultParams[paramName])`?

Reply:
Fixed